doc/manual.rst

```diff
@@ -2843,6 +2843,35 @@ expanded into a list of its elements:
     of '0'..'9': echo "a number"
     else: echo "other"
 
+The ``case`` statement doesn't produce an l-value, so the following example
+won't work:
+
+.. code-block:: nim
+  type
+    Foo = ref object
+      x: seq[string]
+
+  proc get_x(x: Foo): var seq[string] =
+    # doesn't work
+    case true
```

Comment:
Is `case true` even valid? It's like saying `case 123` for ints.

Reply:
huh, I am surprised that this is valid: https://play.nim-lang.org/#ix=2b6b

Reply:
why wouldn't it? bool is an ordinal types and case statements work on ordinals.